karma.conf.js

```diff
@@ -27,7 +27,7 @@ var configuration = {
     port: 9876,
     colors: true,
     autoWatch: true,
-    browsers: ['PhantomJS'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Try something like that:
`    browsers: ['ChromeDebug'],
    customLaunchers: {
      ChromeDebug: {
        base: 'ChromeHeadless',
        flags: ['--no-sandbox', '--disable-setuid-sandbox', '--remote-debugging-port=9222', 'http://localhost:9876/debug.html']
      }
    },`

You'll be able to use Chrome Debugger in VS Code using this task too if you do so:
`    {
      "type": "chrome",
      "request": "attach",
      "name": "Attach VS Code debugger to Chrome debugger",
      "port": 9222,
      "webRoot": "${workspaceRoot}",
      "url": "http://localhost:9876/debug.html#",
    },`